modules/claims_api/lib/bd/bd.rb

```diff
@@ -64,29 +64,28 @@ def upload(claim:, pdf_path:, doc_type: 'L122', file_number: nil, original_filen
     # @return {parameters, file}
     def generate_upload_body(claim:, doc_type:, pdf_path:, file_number: nil, original_filename: nil)
       payload = {}
-      veteran_name = "#{claim.auth_headers['va_eauth_firstName']}_#{claim.auth_headers['va_eauth_lastName']}"
-      file_name = generate_file_name(doc_type:, veteran_name:, claim_id: claim.evss_id, original_filename:)
+      auth_headers = claim.auth_headers
+      veteran_name = "#{auth_headers['va_eauth_firstName']}_#{auth_headers['va_eauth_lastName']}"
+      birls_file_num = auth_headers['va_eauth_birlsfilenumber'] || file_number
+      claim_id = doc_type == 'L705' ? claim.claim_id : claim.evss_id
+      file_name = generate_file_name(doc_type:, veteran_name:, claim_id:, original_filename:)
+      participant_id = auth_headers['va_eauth_pid']
```

Comment:
we only want one or the other here (file number or participant ID). This will try to assign both to the body for the call to BD

Reply:
@FonzMP I added conditionals for the participant_id and file_num based on doc_type